jclouds-shaded/pom.xml

```diff
@@ -44,6 +44,19 @@
       <groupId>javax.annotation</groupId>
       <artifactId>javax.annotation-api</artifactId>
     </dependency>
+    <!-- https://github.com/apache/jclouds/pull/123/files-->
+    <dependency>
+      <groupId>com.google.inject</groupId>
+      <artifactId>guice</artifactId>
+      <version>${guice.version}</version>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.inject.extensions</groupId>
+      <artifactId>guice-assistedinject</artifactId>
+      <version>${guice.version}</version>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
Would it be useful to add these to the root parent pom.xml's dependencyManagement / dependencies so that the correct versions get picked in all locations?